backend/tracim_backend/models/context_models.py

```diff
@@ -496,6 +496,16 @@ def __init__(
         self.limit = limit
 
 
+class KnownContentsQuery(object):
+    """
+    Content autocomplete query model
```

Comment:
This kind of explanation seems useless. I suggest to rename the clss in order to have something auto-explicit. Instead of naming the class `KnownContentsQuery` with a docstring almost similar, why not name the class itself to `KnownContentAutocompleteQueryModel`?

Reply:
I am doing whatever is consistent with the surrounding code. I would not have added this doc string if not for the class just before doing this too. I would not rename the class though. It perfectly matches the endpoint's path.